deltachat-ios/Controller/WebxdcViewController.swift

```diff
@@ -392,9 +392,28 @@ extension WebxdcViewController: WKURLSchemeHandler {
                 data = dcMsg.getWebxdcBlob(filename: file)
             }
             let mimeType = DcUtils.getMimeTypeForPath(path: file)
-            let response = URLResponse(url: url, mimeType: mimeType, expectedContentLength: data.count, textEncodingName: nil)
-            
-            urlSchemeTask.didReceive(response)
+            if !data.isEmpty {
+                urlSchemeTask.didReceive(HTTPURLResponse(
+                    url: url,
+                    statusCode: 200,
+                    httpVersion: "HTTP/1.1",
+                    headerFields: [
+                        "Content-Type": mimeType,
+                        "Content-Length": "\(data.count)"
+                    ]
+                )!)
```

Comment:
we normally try to program a little bit more defensively, avoiding force unwrapping. Is it assured this assertion is always true? 
I would propably move the HTTPURLResponse into a `if let` statement